libs/backendless.js

```diff
@@ -929,7 +916,7 @@
 
                 if (responder != null) {
                     isAsync = true;
-                    responder = this._wrapAsync(responder);
+                    responder = wrapAsync(responder, this._parseResponse.bind(this));
```

Comment:
you should not use `bind` function to preserve old browsers support. pass optional `context` as third argument